From 7a6a03b8752679f56c72f65690dd5fb415c6cdf5 Mon Sep 17 00:00:00 2001
From: Anton Mikanovich <amikan@ilbers.de>
Date: Tue, 23 Aug 2022 14:21:26 +0200
Subject: [PATCH 1/3] ubuntu: add initramfs-tools dependency

Package initramfs-tools used for initrd generation was moved to recommended in
linux kernel package of Ubuntu repo, so add it as dependency for rootfs.

Signed-off-by: Anton Mikanovich <amikan@ilbers.de>
---
 meta-isar/conf/distro/ubuntu-focal.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meta-isar/conf/distro/ubuntu-focal.conf b/meta-isar/conf/distro/ubuntu-focal.conf
index 4dfa2014..c0d12967 100644
--- a/meta-isar/conf/distro/ubuntu-focal.conf
+++ b/meta-isar/conf/distro/ubuntu-focal.conf
@@ -23,5 +23,6 @@ DISTRO_KERNELS ?= " \
     snapdragon generic-lpae"
 
 IMAGE_PREINSTALL += "init"
+IMAGE_PREINSTALL += "initramfs-tools"
 
 WIC_IMAGER_INSTALL += "python3-distutils"
-- 
2.39.5

